docs/release-notes/rl-0.7.md

```diff
@@ -285,6 +285,7 @@ To migrate to `nixfmt`, simply change `vim.languages.nix.format.type` to
 [Soliprem](https://github.com/Soliprem):
 
 - Add LSP and Treesitter support for R under `vim.languages.R`.
+  - Add formatter suppoort for R
```

Comment:
Maybe you could mention the packages used for "formatter support" here, sounds too vague otherwise.